### ISSUE TITLE
Kidney volume calculation

### ISSUE THREAD

**Issue by Jesminna** [Open-Systems-Pharmacology/Forum#360]
Hello,

Based on the information in the literature (https://www.ncbi.nlm.nih.gov/pubmed/26323410), it looks like the attached equation is used in PKSim for GFR calculation. Is there any equation used in PKSim for kidney volume calculation? If yes, could you share the equation?

Thanks!

Jesminna
![GFR equation](https://user-images.githubusercontent.com/52975776/61321619-e5673980-a7d9-11e9-8d07-8cad0566fc92.jpg)

**Comment by PavelBal** [comment]
Hello Jesminna,

you can see all the equations if you export the model to MoBi:

![image](https://user-images.githubusercontent.com/25903040/61321953-eb2a4280-a80c-11e9-9180-434afb2e21bf.png)

There you will find a tree of the whole organism in the **Spatial Stuctures** Building Block, where you can see how the parameters are defined.

![image](https://user-images.githubusercontent.com/25903040/61322159-5bd15f00-a80d-11e9-8c6c-52bba7fcab9b.png)


The volume of the kidney is a distributed parameter with a mean value and standard deviation. For a mean individual, the volume is taken from the PK-Sim database according to the anthropomentry you selected for the indiidual.

**Comment by Jesminna** [comment]
Thank you! I found a few equations in literature which was used for kidney volume calculation based on the body weight or kidney weight. The calculated kidney volume is smaller than the data provided in PKSim. Could you share the literature which PKSim uses to generate the mean individual kidney volume data.  Thanks!

**Comment by msevestre** [comment]
@Jesminna 

> Could you share the literature which PKSim uses to generate the mean individual kidney volume data. 

This depends on the population selected when creating the individual

![image](https://user-images.githubusercontent.com/1041237/61340157-1c0b7700-a80f-11e9-9b10-fa0a42eebfaa.png)

Let me know if that helps,
Michael

**Comment by Jesminna** [comment]
I would like to know the reference for preterm infants. Thanks!

**Comment by Incei** [comment]
> 
> 
> I would like to know the reference for preterm infants. Thanks!

Here is the reference link: https://www.ncbi.nlm.nih.gov/pubmed/26323410
Cheers

**Comment by Incei** [comment]
@Yuri05 @msevestre  it may be a good idea to add (Claassen 2015) behind the 'Preterm' population.

**Comment by Jesminna** [comment]
This reference (Claassen 2015) is the one I mentioned in my original question. The following is the data source listed in this article for kidney volume. However, I cannot find any kidney volume information in the following cited article.

[cid:image001.png@01D53C8C.8863FD80]

**Comment by Incei** [comment]
@Jesminna I cannot see the image, which cited article do you refer to? 

In case you mean reference 35 (Basic anatomical and physiological data for use in radiological protection: reference values: ICRP Publication 89: Approved by the Commission in September 2001), information on kidney volume is mentioned there.. 
Based on the ICRP data, an adult pbpk model for Europeans (ICRP2002) was created and kidney volume was calculated and reported in: [https://www.ncbi.nlm.nih.gov/pubmed/17431751](https://www.ncbi.nlm.nih.gov/pubmed/17431751) : Development of a Physiology-Based Whole-Body Population Model for Assessing the Influence of Individual Variability on the Pharmacokinetics of Drugs

This is the (rounded) adult 440mg kidney volume where the Claassen 2015 et al refers to for the modified GFR function.

Hope this helps!

**Comment by Yuri05** [comment]
I cannot see the image in your last comment (when attaching images or files, please do it **directly on GitHub**. If you answer to an **notification email** - attachments are lost).

But I guess you refer to this part of  (Claassen 2015)?
![grafik](https://user-images.githubusercontent.com/25061876/61387554-a529b900-a8b6-11e9-9625-a7c7968919fb.png)

If so, you will find reference for kidney **weights** on page 6(261) of the [referenced article](https://academic.oup.com/ajcp/article/126/2/256/1760149)
We assume `density=1`, thus `volume(kidney)=weight(kidney)`.

**Comment by Jesminna** [comment]
Thank you! I appreciate it. Based on the data in fig 11 in the reference article (Quantitative Standards for Fetal and Neonatal Autopsy), it seems that the kidney volume values are smaller than those provided in PKSim. For example, the kidney volume in fig 11 for a  40 weeks PMA baby is around 25 mL (25g), whereas the kidney volume provided in PKSim for a  40 weeks PMA baby  is 38.6 mL.  The calculated GFR would be around 1.89 (based on a kidney volume of 25 mL) and 3.52 (based on a kidney volume of 38.6) ml/min.

**Comment by Jesminna** [comment]
Any comments on the difference between literature reported and PKSim provided kidney volume data? or is there any other factor that needs to be considered? Thanks!

**Comment by JanSchlender** [comment]
Hi Jasmina,
Your last post was already pointing in the right direction. Within the method section of Claasen's publication, the conversion from postmortem autopsy measures to perfused organ sizes is described and referenced ("Furthermore, all organ weights summarized in this article refer to exsanguinated organs and require correction for their blood content, as already described by Edginton et al. in order to obtain the total in vivo organ weight [Edginton AN, Schmitt W, Willmann S. Development and evaluation of a generic physiologically based pharmacokinetic model for children. Clin Pharmacokinet 2006; 45(10): 1013-34]"). In short, a vascular fraction is added to the autopsy weight. Literature sources for these fractions can be found in the aforementioned source. 
The GFR, resulting from from the perfused kidney, is then well in line with the reported literature for this parameter.
Best,
Jan

**Comment by Jesminna** [comment]
Thanks a lot, Jan, for the detailed information! I was trying to calculate the kidney volume by adding the blood content to the kidney weight, however, I still cannot get the kidney volume data provided in the PKSim. Could you please help with this calculation? The following is my calculation.

The fraction of vascular space in kidney listed in Table II in Edginton 2006 article is 0.23. Since there is no evidence of age-dependence for the organ vascular space, the blood content in a newborn baby kidneys equals 0.23*270=62.1 mL. The number of 270 is the total blood volume in a newborn baby from Annals of the ICRP 89. Apparently the number of 62.1 mL is too big. Could you help with this calculation?

In addition, the total fraction of vascular space (2.19) showed in Table II in Edginton 2006 article is greater than 1. I do not quite understand it. Could you share your comments on this?
Thanks!

**Comment by prvmalik** [comment]
The vascular fraction of each organ is the fraction of total organ volume attributed to vascular space. 

It is NOT the fraction of total blood volume that is residing in the organ at any given time, as you have explained.

In Table II in Edginton 2006 each organ is assigned a vascular space fraction and an interstitial space fraction (along with relevant age dependencies as defined) such that:

Fv*V + Fi*V + Fc*V = V

where Fv is the vascular fraction
Fi is the interstitial fraction
and Fc is the cellular space fraction

In other words, for every organ, Fv + Fi + Fc always add up to 1.

**Comment by Jesminna** [comment]
On page 1016 in Edginton 2006, there is an explanation about how to obtain the vascular space that "the vascular space of each organ was calculated from its blood content, taken as a percentage of total blood volume in adult". Did I miss anything?

In addition, based on your explanation, can I use the following equation to calculate kidney volume assuming density=1?

Kidney volume=0.23*Kidney volume + Kidney weight

The calculated kidney volume for a 40 weeks PMA baby  is 33 mL. The Kidney volume in PKSim is 38.6 mL.

**Comment by tobiasK2001** [comment]
Dear Jesminna,

I think the calcaltion should like prvmalik pand Jan pointed out more like this:
V(kidney)= 38.6 ml = 0.23(fraction vascular) x 38.6+0.57(fraction intracellular) x 38.6+0.2(fraction interstistial) x 38.6
B.t.w. the vascular and interstitial fraction for each organ in PK-sim can be found here:
![image](https://user-images.githubusercontent.com/25585248/61613024-79b12080-ac60-11e9-8850-32f6f4957188.png)

According to this the "perfused kidney tissue" volume (, without the fraction vascular) would be: 0.57(fraction intracellular) x 38.6+0.2(fraction interstistial) x 38.6 = 0.77 x 38.6= 29.7 ml

Assuming a density = 1 this would give 27.9 g perfused tissue weight. Using that value the GFR should be in range with reported values.

Did that solve your question?

Best, Tobias

**Comment by Jesminna** [comment]
Thank you, Tobias! I have to go back to my original question regarding how to calculate kidney volume or where I can find the kidney volume data. your calculation was based on the known kidney volume.

**Comment by JanSchlender** [comment]
Hi Jesminna,
Since I described it only theoretically last time, let me go through it for this exact case. Kidney volume for term neonates (GA 40weeks) is reported to be V<sub>kid,autopsy</sub>=25ml based on autopsy data. Although different sources were used to inform this exact age by Claasen et al 2015 (based on Archie et al) and Edginton et al 2006 (based on ICRP), the volume is the same in both primary sources. Earlier, I mentioned that a vascular fraction needs to be added to the exsanguinated autopsy weight, which is F<sub>vascular</sub>=0.23 in case of the kidney.
V<sub>vasc,kidney</sub>=V<sub>kid,autopsy</sub> x F<sub>vascular</sub>/(1- F<sub>vascular</sub>)=7.47ml
V<sub>vasc,kidney</sub> + V<sub>kid,autopsy</sub>=32.47ml 
Since PK-Sim relies on a whole-body PBPK model in which all organs add up to the total bodyweight, the remaining difference between reported total age-specific bodyweight and the sum of all organs for the same age is distributed proportionally to all organs. This difference is caused by unconsidered organs in the whole-body framework and is less than 10% for each age-step. Brain, scaled absolutely, and Fat, accounts for over- or underweight, are not altered in this step based on its scaling characteristics described in Willmann et al 2007 (https://www.ncbi.nlm.nih.gov/pubmed/17431751). 
The advantage of this procedure is to avoid an un-physiological “rest-container” which my even change the trend of its size and perfusion multiple times across the entire age-span. 
Let me explain the calculation to account for this gap at the specific age you are interested in:
•         Sum of all Organs considered in the PK-Sim PBPK framework including specific V<sub>vasc</sub> and blood pools except Fat and Brain=1853.94ml (V<sub>Sum,Organs</sub>)
•         Percentage Kidney of all Organs=32.47ml/1853.94ml x 100=1.75%
•         V<sub>Kid,PK-Sim</sub>=1.75%/100 x (3500ml [Target BW] – 396.66ml [V<sub>Brain</sub>] – 906.31ml [V<sub>Fat</sub>])=38.48ml (Deviations in the last digit due to rounding for this explanation)
Best,
Jan

**Comment by Jesminna** [comment]
Hi Jan,
Thanks very much for your detailed explanation! First of all, I can understand that the kidney volume (kidney weight) based on the autopsy data needs correction for their blood content.  For a term neonate (GA 40 weeks), the calculated kidney volume after corrected with blood content is 32.47 ml. Then you mentioned that this number 32.47 ml needs further correction due to the difference between the total bodyweight value in PKSim and the reported age-specific bodyweight. I have a few questions regarding your calculations and would appreciate if you could help.
1.	The calculated kidney volume (32.47 mL) is based on the literature data. I think this number is not related to any parameter value used in PKSim. My understanding is it doesn’t matter how PKSim gets the total bodyweight, and it should not affect the calculated kidney volume based on the literature data. Please correct me if I am wrong.
2.	You mentioned that “This difference is caused by unconsidered organs in the whole-body framework”, my question is why the difference is distributed proportionally to all organs in PKSim instead of assigning another organ to account for the weight different. Would this cause the deviation of the considered organ weight?
3.	Could you let me know where I can find the total volume of all organs considered in the PK-Sim PBPK framework including specific Vvasc and blood pools except Fat and Brain, which is 1853.94ml (VSum,Organs)? Based on the parameter values in PKSim (attached fig), the calculated total volume (weight) is 3518 mL. The brain volume is 397.4 mL and fat volume is 917.8 mL.
Total volume -Brain volume- Fat volume=3518-397.4-917.8=2203 mL.
4.	Another question is that the calculation of total volume of all organs using PKSim data is based on the known kidney volume. If I do not know kidney volume, how can I obtain the total volume of all organs?
Thanks!

**Comment by Jesminna** [comment]
![organ weight](https://user-images.githubusercontent.com/52975776/64275957-c13ff300-cf14-11e9-8ff8-81951a03f629.jpg)

Here is the fig.

**Comment by JanSchlender** [comment]
Hi Jasmina,
Truly some important observations.

1. As long as you select the mean individual for a given age, the organ volume is not dependent on any other parameter. Since the organ volume is not calculated by a function over age but retrieved from the PK-Sim database described by a mean and a distribution, the “create individual” algorithm identifies a typical individual as the maximum likelihood individual in line with the requested demographic properties (-> Input) by means of numerical maximization of the likelihood function. This step is also summarized in the OSP wiki https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/wiki/Create-Individual-Algorithm  
2. The PK-Sim PBPK framework considers the most relevant organs and tissues in the human body. Unconsidered organs or parts of the body in this framework are e.g. hair, teeth, tooth, nails or tongue. These parts account for 5-10% of the body weight and cause this deviation between the sum of all organs and the body weight. One could either introduce a “rest” compartment which is not physiologically motivated or add this rest proportionally to the other organs. By this the rest obtains average characteristics of the other organs.
3. V<sub>Sum,Organs</sub> is not shown in PK-Sim since this is an intermediate step within the calculation to inform the underlying database before the remaining difference is proportionally distributed to all organs. The slight deviation for total body weight, brain and fat which you observed in PK-Sim compared to the respective database values I stated are due to the optimization step mentioned above. Btw, this optimization step has no random effect so you will always get the same values if you recreate this individual. 
4. For the population you used in PK-Sim (European-ICRP, preterms) kidney volume was informed from literature. Arguably this is sometimes not the case once you investigate certain diseased or special population groups. For this I cannot purpose a gold standard since this should be assessed on a case-by-case basis in order to identify options how the pathophysiology might be informed.

Hope this helps. 
Best,
Jan

**Comment by Jesminna** [comment]
Thanks a lot for your detailed explanation, Jan! That really helps a lot!
